code/__HELPERS/unsorted.dm

```diff
@@ -1413,3 +1413,21 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 //checks if a turf is in the planet z list.
 /proc/turf_z_is_planet(turf/T)
 	return GLOB.z_is_planet["[T.z]"]
+
+//returns a GUID like identifier (using a mostly made up record format)
+//guids are not on their own suitable for access or security tokens, as most of their bits are predictable.
+//	(But may make a nice salt to one)
+/proc/GUID()
```

Comment:
Minor note, I made a GUID generating function purely because I needed something to put in the `window` name for the browse() call.

This generates a standards complaint `{xxxxxxxx-xxxx-mxxx-vxxx-xxxxxxxxxxxx}` Globally Unique IDentifer, using our own record format (since byond doesn't give us time info in a way that made it easy to stick to the version 1 variant a record format.)